### PR TITLE
Fixed unwanted trailing tab in chrome 53

### PR DESCRIPTION
**Pull request by adorin**
Closes https://github.com/minkphp/MinkSelenium2Driver/issues/243

### CODE REVIEW

File: src/Selenium2Driver.php [hunk 1]
```diff
@@ -676,12 +676,11 @@ public function setValue($xpath, $value)
...
         if (in_array($elementName, array('input', 'textarea'))) {
             $existingValueLength = strlen($element->attribute('value'));
-            // Add the TAB key to ensure we unfocus the field as browsers are triggering the change event only
-            // after leaving the field.
-            $value = str_repeat(Key::BACKSPACE . Key::DELETE, $existingValueLength) . $value . Key::TAB;
+            $value = str_repeat(Key::BACKSPACE . Key::DELETE, $existingValueLength) . $value;
         }
 
         $element->postValue(array('value' => array($value)));
+        $this->blur($xpath);
```
Comment (aik099) [inline src/Selenium2Driver.php]: The `change` event should be fired instead of `blur` to make tests pass.

**Comment by aik099** [comment]
Also don't send PR from `master` branch, because you'll be unable to send 2 PR in parallel to same project.

File: src/Selenium2Driver.php (hunk 1 above)
Comment (adorin) [inline src/Selenium2Driver.php]: Done

**Comment by aik099** [comment]
One test still fails. However I'm unsure if this is related to made changes or just to Firefox version updated on Travis CI, that doesn't play nice with Selenium Driver version that is used.

**Comment by aik099** [comment]
For me latest stable Selenium Driver (2.53.0) + latest stable Firefox (48.0.1) doesn't work at all. When running test suite on Chrome I'm getting same `&quot;` error.

**Comment by aik099** [comment]
I've created #245 issue for that failing test because it's not related.

**Comment by aik099** [comment]
Otherwise change looks good if it solves problem you're having in both Chrome and Firefox.

**Comment by sumthief** [comment]
Faced with the same problem on my current project. This solution works for me in Chrome 53.0.2785.116 (64-bit).

**Comment by aik099** [comment]
The build failure is due different problem described in #245.

Otherwise tests are passing and I think this one is ready to be merged.

**Comment by aik099** [comment]
Thanks @adorin , merging.

**Pull request merged**